basedosdados_api/account/models.py

```diff
@@ -22,17 +22,83 @@ class Meta:
 
 
 class Profile(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
     user = models.OneToOneField(User, on_delete=models.CASCADE)
     organizations = models.ManyToManyField(
         Organization, related_name="users", blank=True
     )
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
     name = models.CharField(max_length=255)
+    description = models.TextField(null=True, blank=True)
     birth_date = models.DateField(null=True, blank=True)
     picture_url = models.URLField(null=True, blank=True)
     twitter = models.CharField(max_length=255, null=True, blank=True)
     linkedin = models.CharField(max_length=255, null=True, blank=True)
     github = models.CharField(max_length=255, null=True, blank=True)
     website = models.URLField(null=True, blank=True)
     email = models.EmailField(null=True, blank=True)
+    picture = models.ImageField(upload_to="profile_pictures", null=True, blank=True)
```

Comment:
Esse `upload_to="profile_pictures"` aqui salva em disco? Pq se for o caso, temos que fazer um armazenamento persistente disso no cluster com um PVC

Reply:
Acho que vale já termos isso pronto, ainda que optemos por não usar num primeiro momento.

Reply:
Acho melhor seguir com essa abordagem ao invés de manter isso em um storage remoto, o que acha?